libs/math/include/math/matrix_operations.hpp

```diff
@@ -274,6 +262,14 @@ T Product(std::vector<T> const &obj1)
 template <typename ArrayType>
 meta::IfIsMathArray<ArrayType, void> Max(ArrayType const &array, typename ArrayType::Type &ret)
 {
+  // if (array.size() >= array.data().padded_size())
+  // {
+  //   ret = array.data().in_parallel().Reduce(
+  //       [](auto const &a, auto const &b) { return fetch::vectorise::Max(a, b); },
+  //       [](auto const &a) { return fetch::vectorise::Max(a); });
+  // }
+  // else
+  // {
```

Comment:
do we still need the commented code?

Reply:
the commented code is there until I get the vectorisations for other types as well, then it should work.